flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/source/enumerator/IncrementalSourceEnumerator.java

```diff
@@ -171,6 +206,7 @@ private void assignSplits() {
                 LOG.info("Assign split {} to subtask {}", sourceSplit, nextAwaiting);
             } else {
                 // there is no available splits by now, skip assigning
+                wakeupBinlogReaderIfNeed();
```

Comment:
```suggestion
                wakeupStreamReaderIfNeed();
```

Reply:
Addressed in https://github.com/ververica/flink-cdc-connectors/pull/1838/commits/7490d9b9785adb1db065b68cd2305458e46920aa